apps/prairielearn/python/prairielearn/conversion_utils.py

```diff
@@ -883,6 +920,11 @@ def latex_from_2darray(
 def string_partition_first_interval(
     s: str, left: str = "[", right: str = "]"
 ) -> tuple[str, str, str]:
+    """Split a string at the first occurrence of left and right delimiters.
+
+    Returns:
+        A tuple
```

Comment:
A tuple of?

Reply:
I disabled requiring `Returns` for one-line functions.